wpilib/wpilib/driverstation.py

```diff
@@ -498,22 +490,16 @@ def getLocation(self):
             return 3
         else:
             return 0
-    
-    def waitForData(self, timeout = None):
+
+    def waitForData(self, timeout = 0):
```

Comment:
This actually should be None. 

https://docs.python.org/3.6/library/threading.html#threading.Condition.wait

Reply:
Oh, oops, fixed.